src/entrypoints/options/pages/general/index.tsx

```diff
@@ -1,12 +1,60 @@
+/// <reference types="chrome" />
+
+import { toast } from 'sonner' // 如果项目用 toast 通知
+import {
+  AlertDialog,
+  AlertDialogAction,
+  AlertDialogCancel,
+  AlertDialogContent,
+  AlertDialogDescription,
+  AlertDialogFooter,
+  AlertDialogHeader,
+  AlertDialogTitle,
+  AlertDialogTrigger,
+} from '@/components/ui/alert-dialog'
+import { Button } from '@/components/ui/button'
 import { PageLayout } from '../../components/page-layout'
 import { ReadConfig } from './read-config'
 import TranslationConfig from './translation-config'
 
+function resetToDefaultConfig() {
```

Comment:
we should only reset the `config`, not all the storage in local, and need to use the WXT browser API not chrome